drivers/SmartThings/zigbee-sensor/fingerprints.yml

```diff
@@ -0,0 +1,8 @@
+zigbeeGeneric:
+  - id: "ias-generic-sensor"
+    deviceLabel: "IAS Zigbee Generic Sensor"
```

Comment:
The label is a little bit diffcult. How about the 'Zigbee Safety Sensor"
@greens  Do you have any good idea?